presto-spi/src/main/java/com/facebook/presto/spi/block/FixedWidthBlock.java

```diff
@@ -76,20 +91,22 @@ public int getPositionCount()
     @Override
     public long getSizeInBytes()
     {
-        return getRawSlice().length() + (long) valueIsNull.length();
+        return (fixedSize + Byte.BYTES) * (long) positionCount;
```

Comment:
Extract to a separate commit, since adhering to `getSizeInBytes` contract has nothing to do with eliding the `valueIsNull` array